includes/prompt.h

```diff
@@ -0,0 +1,27 @@
+#ifndef PROMPT_H
+# define PROMPT_H
+
+#include "libft.h"
+#include "directory.h"
```

Comment:
dont include directory.h in header, it is not part of api and it doesnt provide necessary datatypes